Python/formatter_unicode.c

```diff
@@ -260,7 +262,26 @@ parse_internal_render_format_spec(PyObject *obj,
             /* Overflow error. Exception already set. */
             return 0;
 
-        /* Not having a precision after a dot is an error. */
+        if (end-pos && READ_spec(pos) == '_') {
+            if (consumed == 0) {
+                format->precision = -1;
+            }
+            format->frac_thousands_separator = LT_UNDERSCORE_LOCALE;
+            ++pos;
+            ++consumed;
+        }
+        if (end-pos && READ_spec(pos) == ','
+            && format->frac_thousands_separator == LT_NO_LOCALE)
```

Comment:
Should we use a more explicit error message for `._,f` and `.,_f` like for `_,f` and `,_f` above?

Reply:
Probably I could provide more enhanced error message, that make distinction between separators *before* and *after* the dot.  But I'm not sure if this worth it.